ext/handlers/callbackquery.go

```diff
@@ -43,6 +48,6 @@ func (cb CallbackQuery) CheckUpdate(b *gotgbot.Bot, ctx *ext.Context) bool {
 	return cb.Filter == nil || cb.Filter(ctx.CallbackQuery)
 }
 
-func (cb CallbackQuery) Name() string {
-	return fmt.Sprintf("inlinequery_%p", cb.Response)
+func (cb *CallbackQuery) Name() string {
+	return fmt.Sprintf("callback_query_handler_%p", cb.Response)
```

Comment:
Good catch here, thanks for finding it.

Reply:
Ok i will pr this seperate by in the evening due to iam in college now